certbot/docs/install.rst

```diff
@@ -243,6 +243,8 @@ Certbot-Auto
 
 We used to have a shell script named ``certbot-auto`` to help people install
 Certbot on UNIX operating systems, however, this script is no longer supported.
+If you want to uninstall ``certbot-auto``, you can follow our instructions
+:doc:`here <uninstall>`.
```

Comment:
What's the reason for using a separate page, as opposed to inlining it here?

If it's planned to add other distribution methods, then it might be good to add it as a menu item (though we could do this later).

Or is it just because it'll take too much space on `/install`?

Reply:
I think it's largely just a stylistic preference, but it was because I imagined we'd expand the doc in the future to describe other aspects of uninstalling Certbot to resolve https://github.com/certbot/certbot/issues/1245 and I think having any significant docs about that on the page titled "Get Certbot" is a little awkward.

At the same time, I didn't really want to link the doc in the table of contents now because I personally don't think there is enough content there to justify it.